hack/build-kubemacpool-podman.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -z "$ARCH" ] || [ -z "$PLATFORMS" ] || [ -z "$KUBEMACPOOL_IMAGE_TAGGED_1" ]; then
```

Comment:
I'm missing KUBEMACPOOL_IMAGE_TAGGED_2 in the podman script. Is this intentional? 
I would want both tag to be there. having both `latest` and sha-specific [tags](https://quay.io/repository/kubevirt/kubemacpool?tab=tags&tag=latest) allow us to be flexible picking the desired image when deployed from other operators like CNAO.

Reply:
Yes, both tags are properly handled.
For multi-architecture support, we create a single manifest and tag it with both the git_sha_tag and the latest tag before pushing. This process, managed in the Makefile, ensures that both tags are properly handled.